src/reducers/index.js

```diff
@@ -47,6 +47,8 @@ const initialState = {
   addressesFound: 0,
   // Quantity of transactions already loaded to give a feedback to the user
   transactionsFound: 0,
+  // Height of the network arrived from ws data
+  height: 0,
```

Comment:
It seems we are not using it anywhere in the PR.

Reply:
We need it on redux so it will call `mapStateToProps` when it changes, so I can calculate the balance again.

Reply:
Ok. Thanks!